Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -60,14 +58,34 @@ static inline CGPoint __CreateCGPointWithTransform(CGFloat x, CGFloat y, const C
 };
 
 struct __CGPath : CoreFoundation::CppBase<__CGPath, __CGPathImpl> {
+    inline ComPtr<ID2D1PathGeometry>& PathGeometry() {
+        return _impl.pathGeometry;
+    }
+
+    inline ComPtr<ID2D1GeometrySink>& GeometrySink() {
+        return _impl.geometrySink;
+    }
+
+    inline bool& IsFigureClosed() {
+        return _impl.isFigureClosed;
+    }
+
+    inline CGPoint& CurrentPoint() {
+        return _impl.currentPoint;
+    }
+
+    inline CGPoint& StartingPoint() {
+        return _impl.startingPoint;
+    }
+
     // A private helper function for re-opening a path geometry. CGPath does not
     // have a concept of an open and a closed path but D2D relies on it. A
     // path/sink cannot be read from while the path is open thus it must be
     // closed. However, a CGPath can be edited again after being read from so
     // we must open the path again. This cannot be done normally, so we must
     // create a new path with the old path information to edit.
     void preparePathForEditing() {
```

Comment:
I don't recall doing that, but I think naming consistency within a class trumps other concerns. :)

---
In reply to: [85790869](https://github.com/Microsoft/WinObjC/pull/1255#discussion_r85790869) [](ancestors = 85790869,85629662)